esp-link/main.c

```diff
@@ -79,7 +79,19 @@ static void ICACHE_FLASH_ATTR prHeapTimerCb(void *arg) {
 }
 #endif
 
+# define VERS_STR_STR(V) #V
+# define VERS_STR(V) VERS_STR_STR(V)
+char* esp_link_version = VERS_STR(VERSION);
+
 void user_rf_pre_init(void) {
+  system_set_os_print(DEBUG_SDK);
```

Comment:
this turns off the default esp sdk debugging statements if it's DEBUG_SDK is not defined.